nflow-rest-api-jax-rs/src/test/java/io/nflow/rest/v1/jaxrs/WorkflowInstanceResourceTest.java

```diff
@@ -3,6 +3,7 @@
 import static com.nitorcreations.Matchers.hasField;
 import static io.nflow.engine.workflow.instance.WorkflowInstanceAction.WorkflowActionType.externalChange;
 import static java.util.Arrays.asList;
+import static java.util.Collections.emptySet;
```

Comment:
WorkflowInstanceResourceTest fails on travis-ci.

https://travis-ci.org/NitorCreations/nflow/jobs/533785975

This could be helpful in fixing tests.
`lenient().when(...).thenReturn(..)`

Reply:
Fixed